packages/ecommerce-app-base/src/SkuPicker/SkuPicker.tsx

```diff
@@ -48,6 +58,10 @@ function defaultGetSaveBtnText(selectedSKUs: string[]): string {
   }
 }
 
+function defaultMakeSearchPlaceholderText(_skuType?: string): string {
```

Comment:
Definitely not blocking, but since we have the nice cleanly defined type: `MakeSearchPlaceholderText`, would it make sense to type this function?  Like I said, not blocking, no impact to customers but since we have the type, maybe it would save a future developer 30 seconds?  Not sure.

I could go either way :)